patches/node/fix_override_createjob_in_node_platform.patch

```diff
@@ -0,0 +1,45 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Keeley Hammond <khammond@slack-corp.com>
+Date: Tue, 2 Aug 2022 12:52:02 -0700
+Subject: fix: override createjob in node_platform
+
+This CL changed Platform::CreateJob to an abstract method:
+https://chromium-review.googlesource.com/c/v8/v8/+/3779694
+This patch adds an override for NodePlatform::CreateJob, using
+the same parameters as PostJob.
+
+This patch likely can't be upstreamed, as the CL specifically
+called out embedders needing to to override the method
+themselves.
```

Comment:
Won't Node.js run into this when they update V8..?

Reply:
That's a good point, this could likely be upstreamed to Node.js itself. I'll remove that comment.